RobotCode/motion.py

```diff
@@ -0,0 +1,105 @@
+import turtle
+import math
+import numpy as np
+import time
+import tkinter as tk
+import serial
+import threading
+import struct
+from threading import Thread
+
+global logic
+logic = "test"
+
+# disable_failsafe = 0
+# command1 = struct.pack('<hhhHBH', 20, 0, -20, 0, disable_failsafe, 0xAAAA)
+# OmniMotionRobot.send(motion_irl, command1)
+
+
+class IRobotMotion:
+    def open(self):
+        pass
+    def close(self):
+        pass
+    def move(self, x_speed, y_speed, rot_speed):
+        pass
+
+class OmniMotionRobot(IRobotMotion):
+    def __init__(self):
+         # Wheel angles
+        self.motor_config = [240, 0, 120]
+        self.serialObj = serial.Serial()
```

Comment:
Have naming more unified. Currently the upper row is snake_case and this row is camelCase